Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -446,6 +446,13 @@ define([
     var scratchCartesian = new Cartesian3();
     var scratchCartesianTwo = new Cartesian3();
     var scratchCartesianThree = new Cartesian3();
+    var scratchZoomViewOptions = {
+      orientation: {
```

Comment:
Could this just be as simple as `orientation : new HeadingPitchRoll()` and keep everything else the same and this still works?  We generally don't use object literals when we have explicit types.  Heading/pitch/roll is a bit more awkward since that type was introduced after the separate properties were used in several places.